nuxeo-arender-previewer-apb/templates/arender_previewer_dc.yml.j2

```diff
@@ -130,5 +132,8 @@ spec:
           emptyDir: {}
         - name: work
           emptyDir: {}
+        - name: {{ arender_previewer_config_name }}
+            defaultMode: 420
+           name: config-volume
```

Comment:
That's not valid YAML and should rather  be :
```yaml
        - name: config-volume
          configMap:
            name: {{ arender_previewer_config_name }}
```

Reply:
made changes

Reply:
It has to be valid YAML. The `configMap` has to be aligned with the `name` on the previous line.